### ISSUE TITLE
I found that the initial parameters(kernel_initializer and kernel_regularizer) are more import than octave

### ISSUE THREAD

**Issue by gogo03** [koshian2/OctConv-TFKeras#11]
def _create_normal_residual_block(inputs, ch, N,dropout):
    # Conv with skip connections
    x = inputs
    for i in range(N):
        # adjust channels
        if i == 0:
            skip = Conv2D(ch, 1,kernel_initializer='he_normal',
                          kernel_regularizer=l2(0.0005))(x)
            skip = BatchNormalization(momentum=0.9, epsilon=1e-5)(skip)
            skip = Activation("relu")(skip)
            if dropout>0:
                skip=Dropout(dropout)(skip)
        else:
            skip = x
        x = Conv2D(ch, 3, kernel_initializer='he_normal',
                          kernel_regularizer=l2(0.0005),padding="same")(x)
        x = BatchNormalization(momentum=0.9, epsilon=1e-5)(x)
        x = Activation("relu")(x)
        if dropout>0:
                x=Dropout(dropout)(x)
        x = Conv2D(ch, 3, kernel_initializer='he_normal',
                          kernel_regularizer=l2(0.0005),padding="same")(x)
        x = BatchNormalization(momentum=0.9, epsilon=1e-5)(x)
        x = Activation("relu")(x)
        if dropout>0:
                x=Dropout(dropout)(x)
        x = Add()([x, skip])
    return x

def create_normal_wide_resnet(N=4, k=10,classes=10,input_shape=(32,32,3)):
    """
    Create vanilla conv Wide ResNet (N=4, k=10)
    """
    # input
    input = Input(input_shape)
    dropout=0.0
    # 16 channels block
    x = Conv2D(16, 3, padding="same",strides=(1,1),kernel_initializer='he_normal',
                          kernel_regularizer=l2(0.0005))(input)
    x = BatchNormalization(momentum=0.9, epsilon=1e-5)(x)
    x = Activation("relu")(x)
    x = Dropout(dropout)(x)
    # 1st block
    x = _create_normal_residual_block(x, 16*k, N,dropout)
    # The original wide resnet is stride=2 conv for downsampling,
    # but replace them to average pooling because centers are shifted when octconv
    # 2nd block
    x = AveragePooling2D(2)(x)
    x = _create_normal_residual_block(x, 32*k, N,dropout)
    # 3rd block
    x = AveragePooling2D(2)(x)
    x = _create_normal_residual_block(x, 64*k, N,dropout)
    x = Conv2D(64*k, 1)(x)

    x = BatchNormalization(momentum=0.9, epsilon=1e-5)(x)
    x = Activation("relu")(x)
    
# FC
    x = Flatten()(x)
    x = Dense(classes, activation="softmax",kernel_initializer='zeros',#'he_normal',
        kernel_regularizer=l2(0.0005))(x)
    model = Model(input, x)
    return model

If you use the above codes,you'll find that the accuracy of OctConv  is lower than normal WRN.here OctConv using the same initial paramets like the following codes:
    OctConv2D(filters=ch, alpha=alpha,kernel_initializer='he_normal',
                          kernel_regularizer=l2(0.0005))([high, low])

**Comment by koshian2** [comment]
@gogo03 
Hi, in your case, L2 regularization is probably the most effective.
Have you tried changing the L2 regularization values ​​in various ways?

**Comment by gogo03** [comment]
I have not tried another ways.  Octconv maybe not  a effective method,but it can open our mind.
if i have time,I'll try the dataset of imagenet-1k ,Perhaps the size of the input image has an effect on the result?

**Comment by gogo03** [comment]
because octconv has low Convs,it may loss a lot of features compares with high conv.

**Comment by koshian2** [comment]
@gogo03 
That's a fun experiment. Perhaps it is a problem with hyperparameters rather than the size of the input image.

It's just my thought, but OctConv may be one of the regularization methods. This is because introducing low frequency paths gives blurring or fluctuation to the input image. So you may need to retune hyperparameters like L2 regularization. Please try it if you have time.

**Comment by gogo03** [comment]
Hi,koshian2:
    I have tried the dataset of imagenet_3(imagenet_10)  which selected 3 or 10 categories from imagenet-1k),the input shape is (112,112,3) batch is 8,I found octconv is better than normal on the condition of using L2 regularization,the accuracy is  82.66% and 74.60,respectively. Maybe my thought is right,the input size effect the result.
   If you have better GPUS ot TPU and free time, you can test imagenet-1k and share us the result?OK

**Comment by koshian2** [comment]
@gogo03 
Hi, great work!!
So, OctConv needs enough resolution to work as written in the paper, right?
Choosing a few categories from ImageNet-1k is a nice idea. This is not so difficult training. Thank you so much.
